draft-ietf-quic-ack-frequency.md

```diff
@@ -250,30 +250,29 @@ Ack-Eliciting Threshold:
 
 : A variable-length integer representing the maximum number of ack-eliciting
   packets the recipient of this frame receives before sending an acknowledgment.
-  A receiving endpoint SHOULD send at least one ACK frame when more than this
-  number of ack-eliciting packets have been received. A value of 0 results in
+  A receiving endpoint SHOULD send at least one ACK frame after receiving more
+  than this many ack-eliciting packets. A value of 0 results in
   a receiver immediately acknowledging every ack-eliciting packet. By default, an
   endpoint sends an ACK frame for every other ack-eliciting packet, as specified in
   {{Section 13.2.2 of QUIC-TRANSPORT}}, which corresponds to a value of 1.
 
 Requested Max Ack Delay:
 
-: A variable-length integer representing the value to which the endpoint
-  requests the peer update its max_ack_delay
+: A variable-length integer representing the value to which the data sender
+  requests the data receiver update its max_ack_delay
   ({{Section 18.2 of QUIC-TRANSPORT}}). The value of this field is in
   microseconds, unlike the max_ack_delay transport parameter, which is in
   milliseconds. On receipt of a valid value, the endpoint SHOULD update
-  its max_ack_delay to the value provided by the peer. Note that values
-  of 2^14 or greater are invalid for max_ack_delay. A value smaller than
-  the min_ack_delay advertised by the peer is also invalid. Receipt of an
-  invalid value MUST be treated as a connection error of type
-  FRAME_ENCODING_ERROR.
+  its max_ack_delay to the value provided by the peer. Values of 2^14 or
+  greater and values smaller than the min_ack_delay advertised by the peer
+  are invalid. Receipt of an invalid value MUST be treated as a connection error
```

Comment:
There is a very subtile point here which maybe wasn't clear but now is lost entirely. The limit of 2^14 is defined by RFC9000 for max_ack_delay. We don't want to respecify this here (so in case any changes are needed those only need to be done in one place). That's why it was important that it previously said "Note that ...". But maybe it would be good to make that more clear...?

Reply:
Good point, I reverted this portion of the change and added a reference to RFC9000.